src/mumble/MainWindow.cpp

```diff
@@ -715,13 +715,35 @@ void MainWindow::saveImageAs() {
 	if (!ok) {
 		// In case fname did not contain a file extension, try saving with an
 		// explicit format.
-		ok = img.save(fname, "JPG");
+		ok = img.save(fname, "PNG");
 	}
+
+	updateImagePath(fname);
+
 	if (!ok) {
 		g.l->log(Log::Warning, tr("Could not save image: %1").arg(Qt::escape(fname)));
 	}
 }
 
+QString MainWindow::getImagePath(QString filename) const {
+	if (g.s.qsImagePath.isEmpty() || ! QDir::root().exists(g.s.qsImagePath)) {
```

Comment:
```QDir(g.s.qsImagePath).exists()``` would be clearer imho. The way it is written it looks like we somehow have to deal with paths relative to ```C:\```.